perma_web/perma/templates/archive/single-link.html

```diff
@@ -239,12 +239,15 @@
     {% endif %}
 
   </header>
+
 {% endblock %}
 
 {% block mainContent %}
 
   {% if link.user_deleted %}
     {% include "archive/deleted.html" %}
+  {% elif redirect_to_download_view %}
+    {% include "archive/download_to_view_pdf.html" %}
   {% elif can_view %}
     {% include "archive/iframe.html" %}
   {% elif link.is_private %}
```

Comment:
I think maybe this new part needs to be nested as an option inside the "elif can_view" block. As is, does this expose links that people wouldn't be allowed to view, via a laptop/desktop?